src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,14 @@
+body, h1, h2, h3, h4, h5, h6, ul, p {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
```

Comment:
check your scss files, in some places you write some of this style again even if it is already reseted.
try not to dublicate and not to write redundant code